lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'active_support'
+
+module Logidze
+  module VersionedAssociation
+    module SingularAssociation
+      def reader(force_reload = false)
```

Comment:
But we can still use `load_target` for single association too, can't we? I would prefer to stick with as less monkey-patched method as possible. Thus, if we can handle all cases within `load_target`, then let's do that.